dashboard/config/environments/development.rb

```diff
@@ -59,6 +59,7 @@
   # Whether or not to skip script preloading. Setting this to true
   # significantly speeds up server startup time
   config.skip_script_preload = true
+  config.skip_preload_schools = false
```

Comment:
shouldn't this default to `true` in `development` env? Most people running development server won't be working on the auto-complete feature, so the extra ~15 seconds on every start-up would not be ideal in this environment.

Reply:
Sure. I will update